packages/mdc-top-app-bar/README.md

```diff
@@ -0,0 +1,122 @@
+<!--docs:
+title: "Toolbars"
+layout: detail
+section: components
+excerpt: "A container for multiple rows of items such as application title, navigation menu, or tabs."
+iconId: toolbar
+path: /catalog/topappbar/
+-->
+
+# Top App Bars
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/toolbar/index.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/toolbars.png" width="494" alt="Toolbars screenshot">
+  </a>
+</div>-->
+
+MDC Top App Bar acts as a container for containing items such as
+application title, navigation menu, and action items, among other things. Top app bars
+scroll with content by default.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/toolbars.html">Material Design guidelines: Toolbars</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/toolbar/index.html">Demo</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install --save @material/top-app-bar
+```
+
+## Usage
+
+### HTML Structure
+
+Wrap the items with `mdc-top-app-bar` class in the following way:
```

Comment:
I realize this was adapted from the toolbar readme, but this line sounds weird and I'd vote we just remove it. We have other components that forego an opening line here entirely.